bemani/api/objects/catalog.py

```diff
@@ -155,11 +155,30 @@ def __format_jubeat_extras(self) -> Dict[str, List[Dict[str, Any]]]:
         else:
             return {"emblems": []}
 
+    def __format_iidx_extras(self) -> Dict[str, List[Dict[str, Any]]]:
+        # Gotta look up the unlock catalog
+        items = self.data.local.game.get_items(self.game, self.version)
+
+        return {
+            "qpros": [
+                {
+                    "identifier": item.data.get_str("identifier"),
```

Comment:
What's the difference between 'id' and 'identifier'? Those are the same word in English.

Reply:
It's a stripped down filename that is essentially the name of the qpro but in ascii. Maybe ascii_name or just filename would be better but I've already been using this...

Reply:
As long as the documentation is updated (which is is) that's fine. It was not clear in the code and there was no comments.